ci/kind/kind-setup.sh

```diff
@@ -565,6 +565,7 @@ function clean_kind {
       flock -x 200
 
       current_timestamp=$(date +%s)
+      touch ~/.antrea/.clusters.swp
```

Comment:
touch is not ideal as it will not reset contents... this is why I suggested a command that would truncate the file